fvm/env.go

```diff
@@ -214,7 +214,7 @@ func (env *commonEnv) GetValue(owner, key []byte) ([]byte, error) {
 
 	err = env.Meter(meter.ComputationKindGetValue, uint(valueByteSize))
 	if err != nil {
-		return nil, fmt.Errorf("get value failed: %w", err)
+		return nil, fmt.Errorf("meter failed: %w", err)
```

Comment:
the changes in fvm/env.go are changing the diverse error messages into one error message `meter failed`. Is this intentional?